setup.py

```diff
@@ -26,7 +26,7 @@ def run(self):
 
 setup(
     name='django-q',
-    version='1.1.0',
+    version='1.2.0',
```

Comment:
I prefer to keep the versioning out of the PR's. It's never clear when it will be released, there could be some package updates in between for example.